HTML/mockup-mogo/js/renderHTML.js

```diff
@@ -0,0 +1,73 @@
+export const renderBlogs = (parent, typeOfBlog, serverData) => {
+  for (let i = 3; i < 6; i++) {
```

Comment:
magic numbers? :)

render function probably should take list of elements which need to render and don't have some additional logic